gcloud/bigquery/job.py

```diff
@@ -990,10 +1061,13 @@ def _build_resource(self):
             },
             'configuration': {
                 self._JOB_TYPE: {
-                    'query': self.query,
+                    'query': self.query
                 },
             },
         }
+        if self._udf_resources is not None:
+            resource['configuration'][self._JOB_TYPE][self._UDF_KEY] = \
+                _build_udf_resources(self._udf_resources)
```

Comment:
Please no backslash continuations.  This assignment belongs inside `_populate_config_resource` anyway, which would make it simpler.